webapp/lib/view.dart

```diff
@@ -750,6 +768,31 @@ class BatchRepliesConfigurationView extends PackageConfiguratorView {
   }
 }
 
+class ChangeCommunicationsConfigurationView extends PackageConfiguratorView {
+  ChangeCommunicationsConfigurationView(Map<String, model.Configuration> data) : super(data.keys.toList()) {
+    _buildContentPartial(data[controller.selectedPackage]);
```

Comment:
As I mentioned in previous PRs - the view should only get the information it needs to generate it, rather than it needing to make the choice of what to show itself. This means that here we should only pass the name of the selected package, and the configuration data for that specific package, rather than the whole map of all of the package names.

Also - all of the packages have the same structure (at least for now), so actually we don't need different classes for each of them. We can just pass the name of the package and config data to the main `PackageConfiguratorView`.

Reply:
Changed it to only take what it needs.